changes/ 19332-clear-secrets-with-gitops

```diff
@@ -0,0 +1 @@
+Enable gitops to create teams with no enroll secrets, or clear enroll secrets for an existing team by setting team_settings.secret to nothing or to an empty array.
```

Comment:
In this note, is this what you mean by nothing:
```
secrets:
```
?
Is it the same as 
```
secrets: null
```
?

Maybe add "to nothing/null"?

Reply:
Will do another PR for this change.